Xamarin.Forms.Controls/GalleryPages/DynamicViewGallery.cs

```diff
@@ -21,11 +21,12 @@ public class DynamicViewGallery : ContentPage
 						}
 					}, null) },
 				{ nameof(ProgressBar), (() => new ProgressBar(), null) },
+				{ nameof(FloatingActionButton), (() => new FloatingActionButton { ImageSource = "bank.png", HorizontalOptions = LayoutOptions.Center }, null) },
```

Comment:
Testing I found problems changing dynamically the size of the button.
![floatactionbutton-error1](https://user-images.githubusercontent.com/6755973/63167812-ae7f8000-c032-11e9-93c6-2dc48c4f46e5.gif)
When changing any property of the button (for example in the gif, the FlowDirection property) updates the rendering being correct.